Remotion/ObjectBinding/Web.Development.WebTesting.TestSite.Shared/Controls/BocListUserControl.ascx.cs

```diff
@@ -82,6 +83,16 @@ protected override void OnInit (EventArgs e)
                                  };
       JobList_Empty_VariableColumns.AvailableViews.AddRange(variableColumnsViews);
 
+      JobList_NoFakeTableHeader.FixedColumns.Add(
```

Comment:
The other fixed columns are added  in the markup. I think we should stick to that for consistency unless there's a good reason against it.

Reply:
Am not sure I undestand the question. Adding the rowedit-column works because we already do it in other forms.